operators/multiclusterobservability/controllers/placementrule/manifestwork.go

```diff
@@ -598,6 +598,8 @@ func mergeCollectorRuleGroupList(defaultCollectRuleGroupList []operatorconfig.Co
 		}
 	}
 
+	config.CollectRulesEnabled = len(mergedCollectRuleGroups) > 0
```

Comment:
should be this? Any ootb group disabled, don't use the dynamic datasource?
```
config.CollectRulesEnabled = len(mergedCollectRuleGroups) == 2
```

Reply:
Good point. Fixed in the latest commit.